src/controllers/RoleController.ts

```diff
@@ -1,13 +1,18 @@
 import express from "express";
 import { getJobRoles } from "../services/JobRoleService"
+import axios from "axios";
+
+axios.defaults.baseURL = process.env.API_URL || 'http://localhost:3000';
```

Comment:
Let's either pass it in directly or set it globally for the app in your `index.ts` file. The URL variable below seems redundant also

Reply:
I've added the url for localhost:3000 and the AWS frontend into an index.ts file, but I have to keep the base href in the views otherwise it won't redirect. We tested it by changing the port of the URL in the index file to 8080 and it works.